planner/core/task.go

```diff
@@ -676,6 +677,53 @@ func (p *basePhysicalAgg) newPartialAggregate() (partial, final PhysicalPlan) {
 		groupByItems = append(groupByItems, gbyCol)
 	}
 
+	// Optimize
+	// Remove unnecessary schema column.
+	// When the select column is same with the group by key, the column can be removed and gets value from the group by key.
+	// e.g
+	// select a, count(b) from t group by a;
+	// The schema is [firstrow(a), count(b), a]. The column firstrow(a) is unnecessary.
+	// Can optimize the schema to [count(b), a] , and change the index to get value.
+	partialCursor = 0
+	partialAggNum := 0
+	partialAggFuncs := make([]*aggregation.AggFuncDesc, len(p.AggFuncs))
+	partialOptimizeSchema := expression.NewSchema()
+	for i, aggFun := range p.AggFuncs {
```

Comment:
Typically we use `func` to shorten `function`, how about `s/aggFun/aggFunc/`?

Reply:
update.